analyze/timestamp.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Copyright (c) 2018, Linaro Limited
+#
+
+import config
+import io
+from pathlib import Path
+import subprocess
+import sys
+import yaml
+
+from enum import Enum
+
+__all__ = ["Timestamp", "Subsystem", "TimestampParser"]
+
+
+def whoami():
+        return sys._getframe(1).f_code.co_name
+
+
+class Subsystem(Enum):
+    LIBTEEC = 1
+    LINUXMOD = 2
+    OPTEECORE = 3
+
+
+class Addr2Line(object):
+    def __init__(self, binary):
+        if not (isinstance(config.addr2line, Path) and
+                config.addr2line.is_file()):
+            raise SystemError('addr2line binary was not found')
+        if not (isinstance(binary, Path) and binary.is_file()):
+            raise ValueError(str(binary) + " was not found")
+
+        self.process = subprocess.Popen([str(config.addr2line),
+                                        "-e", str(binary)],
+                                        stdin=subprocess.PIPE,
+                                        stdout=subprocess.PIPE)
+        # and let's use buffered input
+        self.stdin_wrapper = io.TextIOWrapper(self.process.stdin, 'utf-8')
+
+    def __del__(self):
+        if self.process:
+            self.process.terminate()
+
+    def lookup(self, addr):
+        try:
+            print(hex(addr))
```

Comment:
Not a big deal, but you are mixing a bit from old (legacy) way of printing and the new (preferred) way using `.format{}`.

Reply:
@jbech-linaro particularly this print was added while debugging (and obviously I forgot to remove it). The same for next `print(info)` on line 54.

Other occurrences will be fixed, thanks!